src/ObjCRuntime/Registrar.cs

```diff
@@ -87,9 +87,9 @@ static class Shared {
 		public static ProductException GetMT4127 (TMethod impl, List<TMethod> ifaceMethods)
 		{
 			var msg = new System.Text.StringBuilder ();
-			msg.Append ("Cannot register more than one interface method for the method '");
+			msg.Append (mtouch.mtouchErrors.MT4127_a);
```

Comment:
This code should be changed to be a single message, for some translations the order of the formatting arguments can change, and that would not be possible to translate like it's implemented now.

Reply:
do the order of arguments being appended in the loop matter?

Reply:
All the English text should be contained within a single string. This means that the text here has to change (the enumeration and string concatenation loop below doesn't work), maybe by creating multiple exceptions:

    MT4127: Cannot register more than one interface method for the method '{0}'. This method implements {0} interface methods.
    MT4???: The method '{0}' implements the interface method '{1}'.
    MT4???: The method '{0}' implements the interface method '{1}'.
    MT4???: The method '{0}' implements the interface method '{1}'.